### DISCUSSION TITLE
Does the latest version of Simu5G fit the Omnetpp 6.1?

### DISCUSSION THREAD

**Discussion by Lance729** [Unipisa/Simu5G#245] [Q&A]
Even though I can deploy the Simu5G on Omnetpp 6.0 as issue #242 said, I wonder if this bug is fixed well. 

It would be better if a pre-release version could be tested. Hopefully.

**Comment by Abh4git** [comment]
For me it works well on 6.0.3.

**Reply by rhornig** [reply]
try the `dev` branch of simu5g. It will eventually pulled into master and then be a base of the next release.

**Reply by Lance729** [reply]
Does the `dev` version of Simu5G fit the latest Omnetpp 6.1?
